### PR TITLE
make constants out of non-changing variables

### PR DESCRIPTION
**Pull request by Godspower-Eze**
Fix build by changing `var` to `const` for constant values as used in current version of zig.

### CODE REVIEW

**Review by gballet — COMMENTED** [review]
build isn't broken, which version of zig are you building with?

**Comment by gballet** [comment]
and your change does break tests:

```
src/main.zig:496:19: error: expected type '*crypto.sha2.Sha2x32(.{.iv0 = 1779033703, .iv1 = 3144134277, .iv2 = 1013904242, .iv3 = 2773480762, .iv4 = 1359893119, .iv5 = 2600822924, .iv6 = 528734635, .iv7 = 1541459225, .digest_bits = 256})', found '*const crypto.sha2.Sha2x32(.{.iv0 = 1779033703, .iv1 = 3144134277, .iv2 = 1013904242, .iv3 = 2773480762, .iv4 = 1359893119, .iv5 = 2600822924, .iv6 = 528734635, .iv7 = 1541459225, .digest_bits = 256})'
src/main.zig:496:19: note: cast discards const qualifier
/opt/hostedtoolcache/zig/0.11.0/x86_64-linux/lib/std/crypto/sha2.zig:115:26: note: parameter type declared here
src/main.zig:496:19: error: expected type '*crypto.sha2.Sha2x32(.{.iv0 = 1779033703, .iv1 = 3144134277, .iv2 = 1013904242, .iv3 = 2773480762, .iv4 = 1359893119, .iv5 = 2600822924, .iv6 = 528734635, .iv7 = 1541459225, .digest_bits = 256})', found '*const crypto.sha2.Sha2x32(.{.iv0 = 1779033703, .iv1 = 3144134277, .iv2 = 1013904242, .iv3 = 2773480762, .iv4 = 1359893119, .iv5 = 2600822924, .iv6 = 528734635, .iv7 = 1541459225, .digest_bits = 256})'
src/main.zig:496:19: note: cast discards const qualifier
/opt/hostedtoolcache/zig/0.11.0/x86_64-linux/lib/std/crypto/sha2.zig:115:26: note: parameter type declared here
```

I'm not against merging this to accommodate future zig versions, but 0.11.0 is the reference version and needs to build.

File: src/main.zig [hunk 1]
```diff
@@ -489,7 +489,7 @@ pub fn merkleize(chunks: []chunk, limit: ?usize, out: *[32]u8) anyerror!void {
             // Merkleize the left side. If the number of chunks
             // isn't enough to fill the entire width, complete
             // with zeroes.
-            var digest = sha256.init(sha256.Options{});
+            const digest = sha256.init(sha256.Options{});
```
Comment (gballet) [inline src/main.zig:492]: so basically, this should still be `var`.

**Comment by Godspower-Eze** [comment]
> build isn't broken, which version of zig are you building with?

0.12.0-dev.1773+8a8fd47d2

**Comment by Godspower-Eze** [comment]
What command do use in running your test?

**Comment by gballet** [comment]
We only support the latest release, so if it breaks in `master` we'll wait for the breaking changes to be merged and released before upgrading versions.

The test commands can be seen in the ci file:

`zig build test --summary all`

**Comment by gballet** [comment]
Could you please remove the .DS_Store?

**Review by gballet — APPROVED** [review]
LGTM. Thank you for being my first contributor!👋

**Pull request merged**